randovania/server/multiplayer/web_api.py

```diff
@@ -170,3 +189,4 @@ def setup_app(sa: ServerApp):
     sa.route_with_user("/session/<session_id>", need_admin=True)(admin_session)
     sa.route_with_user("/session/<session_id>/rdvgame", need_admin=True)(download_session_spoiler)
     sa.route_with_user("/session/<session_id>/delete", methods=["GET", "POST"], need_admin=True)(delete_session)
+    sa.route_with_user("/session/<session_id>/<world_id>/rdvpreset", need_admin=True)(download_world_preset)
```

Comment:
I wanted the route to be more like `/session/<world_id>/rdvpreset`, but when I tried doing that flask complained that session_id was not provided.

Reply:
I would call it `/world/<world_id>/rdvpreset`

Reply:
done